Ascend-Server.Data/Program.cs

```diff
@@ -0,0 +1,8 @@
+
+var builder = WebApplication.CreateBuilder(args);
+
+var app = builder.Build();
+
+app.MapGet("/", () => "Hello World!");
```

Comment:
Your data layer shouldn't be an application that runs or has endpoints. The API project should have your endpoints. Everything else should be class libraries.

Reply:
that means you don't need a program.cs